examples/ea4tx_remotebox.toml

```diff
@@ -25,7 +25,7 @@ name = "My EA4TX Remotebox"
 # switches with an index < 2 and before all other switched with an index > 2.
 index = 2
 portname = "/dev/ttyACM0"
-
-ipaddress = "192.168.10.109"
-ipport = 6000
-typeConnection=1
\ No newline at end of file
+Ipaddress = "192.168.10.109"
```

Comment:
In order to keep things as easy as possible, I suggest to remove the parameters `ipaddress` and `ipport` and use instead the string `portname` for both, the path to the serial device and the IP address. At the end of the day it's just a string. Providing an example for each case in the config file should be clear enough.